tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(5,14): error TS2554: Expected 2 arguments, but got 3.
tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(6,19): error TS2554: Expected 2 arguments, but got 5.
tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(7,19): error TS2556: Expected 2 arguments, but got 4 or more.
tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(8,19): error TS2556: Expected 2 arguments, but got 5 or more.
tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(9,16): error TS2556: Expected 2 arguments, but got 1 or more.
tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts(10,9): error TS2554: Expected 2 arguments, but got 3.


==== tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts (6 errors) ====
    declare function takeTwo(a: string, b: string): void;
    declare const t2: [string, string];
    declare const t3: [string, string, string];
    
    takeTwo('a', ...t2); // error on ...t2
                 ~~~~~
!!! error TS2554: Expected 2 arguments, but got 3.
    takeTwo('a', 'b', 'c', ...t2); // error on 'c' and ...t2
                      ~~~~~~~~~~
!!! error TS2554: Expected 2 arguments, but got 5.
    takeTwo('a', 'b', ...t2, 'c'); // error on ...t2 and 'c'
                      ~~~~~~~~~~
!!! error TS2556: Expected 2 arguments, but got 4 or more.
    takeTwo('a', 'b', 'c', ...t2, 'd'); // error on 'c', ...t2 and 'd'
                      ~~~~~~~~~~~~~~~
!!! error TS2556: Expected 2 arguments, but got 5 or more.
    takeTwo(...t2, 'a'); // error on 'a'
                   ~~~
!!! error TS2556: Expected 2 arguments, but got 1 or more.
!!! related TS6210 tests/cases/conformance/expressions/functionCalls/callWithSpread3.ts:1:37: An argument for 'b' was not provided.
    takeTwo(...t3); // error on ...t3
            ~~~~~
!!! error TS2554: Expected 2 arguments, but got 3.